datadog/metric_test.go

```diff
@@ -351,3 +355,24 @@ func TestCounterStartTime(t *testing.T) {
 		})
 	}
 }
+
+func benchmarkStackdriverData(filename string, b *testing.B) {
```

Comment:
Benchmark tests should be exported.

`func BencmarkStackdriverData(.....`

Reply:
This is a helper function which is called in actual benchmark tests (line 374 onwards) so it is not exported. Should I add a comment in the code to note this?

Reply:
Ah ofcourse - sorry I didn't spot that. All good.